streams/piping/pipe-through.js

```diff
@@ -36,4 +40,55 @@ promise_test(() => {
     assert_array_equals(chunks, [1, 2, 3, 4, 5]), 'chunks should match');
 }, 'Piping through a duck-typed pass-through transform stream should work');
 
+promise_test(t => {
+  const transform = {
+    writable: new WritableStream({
+      start(c) {
+        c.error(error1);
```

Comment:
IIUC, the main purpose of `error1` here is to help identify the source of the problem if the test fails. In which case, wouldn't using a value like `'this rejection should not be reported as unhandled'` be even better?